bridge/engine/EngineStreamDirector.h

```diff
@@ -902,24 +904,24 @@ class EngineStreamDirector
         for (const auto& pinMapEntry : _pinMap)
         {
             auto const& pinnedBy = pinMapEntry.first;
-            auto const& sender = pinMapEntry.second;
-            if (pinnedBy == senderEndpointIdHash || sender != senderEndpointIdHash)
+            auto const& pinTarget = pinMapEntry.second;
+            if (pinnedBy == senderEndpointIdHash || pinTarget != senderEndpointIdHash)
             {
                 continue;
             }
 
             const auto& participant = _participantStreams.find(pinnedBy);
 
             if (participant != _participantStreams.end() &&
-                quality == std::min(participant->second.desiredHighestEstimatedPinnedLevel, highestAvailableQuality))
+                quality == std::min(participant->second.pinQualityLevel, highestAvailableQuality))
```

Comment:
This was also bug that it reported an ssrc to be in use before the quality upgrade has happened. And thus another ssrc reported as not being used.